tasks/main.yml

```diff
@@ -144,4 +144,5 @@
 # pymongo is needed for the mongodb_user Ansible module
 - name: Install pymongo via pip
   ansible.builtin.pip:
+    executable: /usr/bin/pip2
```

Comment:
As a thought should we loop on executablea to install for both Python 2 and 3?

Reply:
Please have a look at commit 029c3c6.

Reply:
Should we hard code the executable for pip3 just to guarantee behavior?

Reply:
I agree with @mcdonnnj here - best to be specific in this case and ensure nothing changes here when the default version for our `ansible-pip-role` moves beyond 3.

Reply:
@mcdonnnj can confirm, but I believe he was referring to hard-coding `pip_path="/usr/bin/pip3"` in the `test_pip3_packages` test, similar to what you did for `pip2`:
https://github.com/cisagov/ansible-role-mongo/blob/034999f8825e8b4a2f5c1a7bacfe1b08c0b6e1d2/molecule/default/tests/test_default.py#L22-L30

Reply:
Please have a look at commit cd76415.